### PR TITLE
Remove unneeded code for dwd deployment

### PR DESCRIPTION
**Pull request by aaronfern**
**How to categorize this PR?**

/area dev-productivity
/kind cleanup

**What this PR does / why we need it**:
Removed unneeded code for dwd deployment 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: pkg/component/nodemanagement/dependencywatchdog/access.go [hunk 1]
```diff
@@ -89,36 +78,6 @@ func (d *dependencyWatchdogAccess) Deploy(ctx context.Context) error {
 	return d.createManagedResource(ctx)
 }
 
-// TODO(aaronfern): Remove this function after v1.93 got released.
```
Comment (ialidzhikov) [inline pkg/component/nodemanagement/dependencywatchdog/access.go:92]: Can also https://github.com/gardener/gardener/blob/e8454f02c43d90c62fb3a7bb4c43919e8d16c971/cmd/gardenlet/app/migration.go#L66 be removed?

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: pkg/component/nodemanagement/dependencywatchdog/access.go (hunk 1 above)
Comment (aaronfern) [reply pkg/component/nodemanagement/dependencywatchdog/access.go:92]: I think you're looking at an outdated branch. This was removed from master in the following PR https://github.com/gardener/gardener/pull/9668

Comment (ialidzhikov) [reply pkg/component/nodemanagement/dependencywatchdog/access.go:92]: Ah, sorry, my bad.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**